int32 KEY_UP = 0
int32 KEY_DOWN = 1
int32 KEY_HOLD = 2

string data
int32 event_type
